unified_planning/model/problem.py

```diff
@@ -331,6 +331,10 @@ def _get_static_and_unused_fluents(
                     unused_fluents.clear()
                     for f in se.fluents:
                         static_fluents.discard(f.fluent())
+            elif isinstance(a, up.model.action.Process):
```

Comment:
What about events here? Again, they are probably absorbed by the `isinstance(a, up.model.action.InstantaneousAction)`, so we need to fix this

Reply:
yep